test/integration/edge-runtime-module-errors/test/utils.js

```diff
@@ -84,7 +90,10 @@ export function expectModuleNotFoundDevError(
   output = context.logs.output
 ) {
   expectModuleNotFoundProdError(moduleName, output)
-  expect(stripAnsi(output)).toContain(importStatement)
+  // turbopack have correct error overlay, but doesn't emit those into cli
+  if (!process.env.TURBOPACK) {
+    expect(stripAnsi(output)).toContain(importStatement)
+  }
```

Comment:
We should fix the problem (emitting to cli) instead of hiding the problem by removing the test check

Reply:
Actually this seems my mistake while looking some other ones. removed.